README.md

```diff
@@ -19,3 +19,74 @@ Github Pages only runs in safe mode, preventing the usage of [symlinks](https://
 1. Copy the contents of `node_modules/@patternfly/patternfly` to `_includes/patternfly`.
 1. Delete `_includes/patternfly/assets` and `_includes/patternfly/icons` as they are not needed.
 1. Commit the changes.
+
+## Deployment
+
+### OKD
+
+1. Copy the generated source in `_site/okd/errors.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/errorpage/templates.go.
+1. Copy the generated source in `_site/okd/login.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/login/templates.go.
+1. Copy the generated source in `_site/okd/providers.html` to the corresponding template at https://github.com/openshift/oauth-server/blob/master/pkg/server/selectprovider/templates.go.
+1. Submit a pull request to https://github.com/openshift/oauth-server containing the copied changes.
+
+To test the changes:
+
```

Comment:
Lets add the `oc` command to each of these: 

`oc create secret generic error --from-file errors.html -n openshift-config` 

People can opt to use the console if they prefer.

Reply:
So i was even thinking the ordering could be swapped to:

- update the jekyll pages & generate the files
- create the secrets in a live cluster to verify that they work
- update the template.go files in openshift/oauth-server and open a PR.

Reply:
I think it's better to tests the source html from the go template as it's possible a mistake is made pasting the html in to the go template.  Otherwise, you're not actually testing what gets put in the oauth-server pr.